core/arch/arm/tee/tee_rpmb.c

```diff
@@ -305,9 +333,12 @@ static TEE_Result tee_rpmb_mac_calc(uint8_t *mac, uint32_t macsize,
 }
 
 struct tee_rpmb_mem {
-	paddr_t phpayload;
+	struct teesmc32_arg *arg;
```

Comment:
OK, I suppose I just need to allocate a `struct teesmc32_param params[2]` on the stack in `tee_rpmb_invoke()`, similar to what is done by `wq_rpc()` in `wait_queue.c` for instance?